tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -615,4 +715,44 @@ public function userChangesOwnPassword(string $user, string $currentPassword, st
 		);
 		$this->featureContext->setResponse($response);
 	}
+
+	/**
+	 * @When user :user gets all the groups using the Graph API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userGetsAllTheGroupsUsingTheGraphApi(string $user): void {
+		$this->featureContext->setResponse($this->listGroups($user));
+	}
+
+	/**
+	 * @When user :user gets all the members of group :group using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $group
+	 *
+	 * @return void
+	 */
+	public function userGetsAllTheMembersOfGroupUsingTheGraphApi($user, $group): void {
+		$this->featureContext->setResponse($this->listGroupMembers($group, $user));
+	}
+
+	/**
+	 * @Then the last response should be an unauthorized response
+	 *
+	 * @return void
+	 */
+	public function theLastResponseShouldBeUnauthorizedReponse(): void {
+		$response = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse());
+		$errorText = $response['error']['message'];
+
+		Assert::assertEquals(
+			'Unauthorized',
+			$errorText,
+			__METHOD__
+			. " Expected text 'Unauthorized' but got '" . $errorText . "'"
```

Comment:
assertions would anyway show that difference. we have to change the wordings here if that got changed. can we just show some message here like:
```suggestion
			. "Expected unauthorized message was found to be changed."
```